engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2519,6 +2519,12 @@ - (void)removeEnableFlutterTextInputViewAccessibilityTimer {
   return _activeView;
 }
 
+- (void)reset {
+  // Reset the text input plugin to prepare for a hot restart.
+  // Hide the software keyboard and text editing context menu if they are visible.
```

Comment:
You already put the same info in `.h` file. duplicate info can get out of sync.

Reply:
Removed!